tests/integration/tokenomics/relay_mining_integration_test.go

```diff
@@ -1,18 +1,21 @@
+//go:build integration
```

Comment:
The rationale for adding the tag here is that these tests will depend on the `ParamsSuite` which is currently using the `integration` build constraint. My thinking was that since the makefile has `test_integration`, running tests shouldn't be problematic. However, I see no reason why we MUST use an `integration` tag and agree that simpler is better. :smile: I will remove them.

I believe that rather than an `integration` tag, it would be more effective to have an `in-memory-network` as those tests are more useful to separate out as they tend to be flaky (leveldb closed, address already in use, etc.).